src/readBuffer.js

```diff
@@ -1,12 +1,19 @@
+const EOF = new Error('EOF')
+exports.EOF = EOF;
```

Comment:
The file uses ES module exports, don’t mix with CommonJS exports

Reply:
I wasn't sure the best way to do that and keep `export default` - other things I tried returned an error. How can I keep `export default function` and also have an EOF variable?

Reply:
You can write

export class Foo {}
export const foo = ...